Beam/resources/assets/js/remplib.js

```diff
@@ -549,6 +575,64 @@ remplib = typeof(remplib) === 'undefined' ? {} : remplib;
             if (0 === this.totalTimeSpent % logInterval) {
                 this.trackTimespent();
             }
+        },
+
+        actualProgress: function() {
+            var h = document.documentElement,
+                b = document.body;
+
+            return (h.scrollTop||b.scrollTop) / ((h.scrollHeight||b.scrollHeight) - h.clientHeight);
+        },
+
+        scrollProgressEvent: function () {
+            var a = this.trackedArticle,
```

Comment:
When called as event callback, `this` isn't pointing to where you might think it's pointing to